### PR TITLE
add recipe 'Coco Loco'

### PR DESCRIPTION
**Pull request by lucasmartinelle**

### CODE REVIEW

File: src/recipes/coco-loco.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        "Add ice cubes.",
+        "Blend for a few seconds.",
+        "Pour the mixture into a large tumbler glass.",
+        "Add a few ice cubes to obtain a fresh cocktail."
+    ],
+    "image": "coco-loco.jpg",
+    "keywords": [
+        "coco",
+        "sugar",
+        "Alcohol",
```
Comment (alfg) [inline src/recipes/coco-loco.json:40]:
```suggestion
        "alcohol",

```

File: src/recipes/coco-loco.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Coco loco",
+    "description": "a rum and coconut cocktail from the West Indies",
```
Comment (alfg) [inline src/recipes/coco-loco.json:3]:
```suggestion
    "description": "A rum and coconut cocktail from the West Indies.",

```

**Review by alfg — CHANGES_REQUESTED** [review]
I think this was branched from your last PR. So I can't merge until the last one is merged/approved. I also added a couple suggestions. Thanks.

**Comment by lucasmartinelle** [comment]
done :)

**Comment by alfg** [comment]
Hi, I think you'll need to sync up this branch with latest from master in order for the updated changeset to show. And to resolve conflict.

**Pull request closed without merging**